rest/route/project.go

```diff
@@ -462,10 +462,7 @@ func (h *projectIDPatchHandler) Run(ctx context.Context) gimlet.Responder {
 	if err = h.sc.UpdateAdminRoles(h.newProjectRef, adminsToAdd, adminsToDelete); err != nil {
 		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "Database error updating admins for project '%s'", h.project))
 	}
-	for i := range h.apiNewProjectRef.Subscriptions {
-		h.apiNewProjectRef.Subscriptions[i].OwnerType = utility.ToStringPtr(string(event.OwnerTypeProject))
-		h.apiNewProjectRef.Subscriptions[i].Owner = utility.ToStringPtr(h.project)
-	}
+
```

Comment:
This is actually handled in SaveSubscriptions

Reply:
in saveProject it only sets the OwnerType to event.OwnerTypeProject if isProjectOwner. And it sets the owner to h.newProjectRef.Id instead of h.project. 
Also, in SaveSubscription it only sets it if certain conditions are met, and here it sets it for everything, potentially leaving something without an owner and ownerType. 

I don't have a full understanding of this, I just want to make sure we are not accidentally overlooking something.

Reply:
We do call SaveSubscriptions with `isProjectOwner=true` on line 468 so I think that's covered. The only other condition I'm seeing for setting it is that the interface converts cleanly (which needs to happen to save the document anyway). Is there something else you're seeing?